app/assets/javascripts/controllers/ops/diagnostics_database_form_controller.js

```diff
@@ -98,6 +108,23 @@ ManageIQ.angular.app.controller('diagnosticsDatabaseFormController', ['$http', '
       (vm.diagnosticsDatabaseModel.log_aws_region === '' || typeof vm.diagnosticsDatabaseModel.log_aws_region === 'undefined'));
   };
 
+  vm.securityProtocolSelect = function() {
+    return vm.diagnosticsDatabaseModel.log_protocol === "OpenStack Swift";
+  };
+
+  vm.swiftSecurityProtocolSelect = function() {
+    return vm.diagnosticsDatabaseModel.log_protocol === 'OpenStack Swift';
+  };
+
+  vm.swiftSecurityProtocolRequired = function() {
+    return (miqDBBackupService.swiftBackup(vm.diagnosticsDatabaseModel) &&
+      (vm.diagnosticsDatabaseModel.security_protocol === null || vm.diagnosticsDatabaseModel.security_protocol === '' || typeof vm.diagnosticsDatabaseModel.security_protocol === 'undefined'));
```

Comment:
`vm.diagnosticsDatabaseModel.security_protocol === null || vm.diagnosticsDatabaseModel.security_protocol === ''`
 is equal to `!vm.diagnosticsDatabaseModel.security_protocol`. 
`typeof vm.diagnosticsDatabaseModel.security_protocol === 'undefined'`
 shouldn't be needed because it's always set to an empty string in `init`.
So `miqDBBackupService.swiftBackup(vm.diagnosticsDatabaseModel) && !vm.diagnosticsDatabaseModel.security_protocol` should be enough.